### DISCUSSION TITLE
Github Copilot Community

### DISCUSSION THREAD

**Discussion by Firoz0006** [community/community#141398] [GitHub Education]
### Hi everyone,

how to get GitHub free copilot for student

**Comment by davevad93** [comment]
Hi @Firoz0006 ,

If you are a student, you can get a free `Copilot` subscription by applying for the `Student Developer Pack` [here](https://education.github.com/discount_requests/application).

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), the requirements to apply to GitHub Education as a student are the following:

- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution
- Have a verifiable school-issued email address or upload documents that prove your current student status
- Have a [GitHub personal account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github)
- Be at least 13 years old

Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.

You can also take a look at the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) in case you encounter any trouble when submitting your application.

**Reply by Firoz0006** [reply]
Thanks

**Reply by davevad93** [reply]
You're welcome @Firoz0006 👍
